tests/foreman/cli/test_host.py

```diff
@@ -73,6 +79,33 @@
 class HostCreateTestCase(CLITestCase):
     """Tests for creating the hosts via CLI."""
 
+    @classmethod
+    def setUpClass(cls):
+        """"""
```

Comment:
Thanks for catching. Of course, leaving the method without docstring is not an option and leaving empty doctring is not acceptable at all. That's something what i left to update later and totally forgot about it.